crawl-ref/docs/macros_guide.txt

```diff
@@ -54,7 +54,7 @@ The macro.txt file
 
 Macros and keymaps are stored in a file called macro.txt in your
 settings/ directory. You can change where the file is read from and
-written to by specifying an alternate directory with the macro_dir
+written to by specifying an alternative directory with the macro_dir
```

Comment:
I would prefer to use "different" instead of "alternative"

Reply:
I've changed that now.